src/core/ColorPicker/Color.tsx

```diff
@@ -0,0 +1,64 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import React from 'react';
+import { useTheme } from '../utils/hooks/useTheme';
+import { getWindow } from '../utils/common';
+import { CommonProps } from '../utils/props';
+import cx from 'classnames';
+import '@itwin/itwinui-css/css/color-picker.css';
+
+export type ColorProps = {
+  /**
+   * Color
+   */
+  color: string;
+  /**
+   * Handler for clicking a color
+   */
+  onColorClicked?: () => void;
```

Comment:
it can be just `onClick`, self explanatory prop and most commonly used.

Reply:
We could also extend `span` props and this won't be needed.

Reply:
I refactored Colors to use extend span props and removed onColorClicked